apps/studio/components/interfaces/DataWarehouse/TestCollectionDialog.tsx

```diff
@@ -78,14 +82,14 @@ export function TestCollectionDialog({
   }
 
   return (
-    <Dialog open={open} onOpenChange={onOpenChange}>
+    <Sheet open={open} onOpenChange={onOpenChange}>
       <Tooltip_Shadcn_>
         <TooltipTrigger_Shadcn_ asChild>
-          <DialogTrigger asChild>
+          <SheetTrigger asChild>
             <Button disabled={accessTokens.length === 0} type="outline">
               Connect
```

Comment:
Any reason why we're calling the CTA "connect"? Instead of something more explicit like "Send event"?
It's a bit off cause i didn't expect this to be the same as "Send test event" from the empty state